app/Models/Foundation/Summit/Summit.php

```diff
@@ -65,6 +65,22 @@ class Summit extends SilverstripeBaseModel
 
     use GetDefaultValueFromConfig;
 
+    public static $default_report_setting = [
+        'scan_date',
+        'attendee_first_name',
+        'attendee_last_name',
+        'attendee_email',
+        'attendee_company',
+        'attendee_extra_questions' => [
```

Comment:
this is not correct
should be attendee_extra_questions and a list of the questions ( id , name) of the available extra question for ticket